mobify/mobify-code-style
# Add jsx-a11y.

Adds accessibility rules for React projects.

## Changes
- Add `eslint-plugin-jsx-a11y`

## How To Test
- Use this branch in a React project
- Add the following image component in one of the `render()` functions: `<img />`
- ESLint should warn you that the image component needs a `alt` prop.

## Applicable Research Resources
- https://github.com/evcohen/eslint-plugin-jsx-a11y

## TODOs:
- [x] Move a11y lint rules into it's own file
